Movie Challenge/Movie Challenge/Controllers/Home/HomeViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  HomeViewController.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 14/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UICollectionViewDataSource, UICollectionViewDelegate{
```

Comment:
To organize better you have to implement the Delegates and DataSources inside extensions blocks.